src/platform/packages/shared/kbn-esql-validation-autocomplete/src/definitions/generated/aggregation_functions.ts

```diff
@@ -1717,26 +1717,6 @@ const valuesDefinition: FunctionDefinition = {
       ],
       returnType: 'boolean',
     },
-    {
-      params: [
-        {
-          name: 'field',
-          type: 'cartesian_point',
-          optional: false,
-        },
-      ],
-      returnType: 'cartesian_point',
-    },
-    {
-      params: [
-        {
-          name: 'field',
-          type: 'cartesian_shape',
-          optional: false,
-        },
-      ],
-      returnType: 'cartesian_shape',
-    },
```

Comment:
This shouldn't be removed. These types were added to `VALUES` last week (see https://github.com/elastic/elasticsearch/pull/122886/files#diff-e173e5cb9b12a03f639b3811c673e267f33cfe0b7928cfafe834ca66f5a67413)

You may have run the generator on an out-of-date Elasticsearch source version

Reply:
Great catch Drew 🤦‍♀️ Done 🙌

Reply:
It would have gotten fixed next Monday anyways 👍